ptf/platform_helper/brcm_t0_sai_helper.py

```diff
@@ -37,6 +38,21 @@ class BrcmT0SaiHelper(CommonSaiHelper):
     platform = 'brcm'
     role_config = 't0'
 
+    # lists of default objects
+    self.port_to_hostif_map = {}
+    self.port_to_bp_map = {}
+    self.hostifs = []
```

Comment:
what's the diff with those two?
    self.hostifs = []

    self.def_hostif_list = []